### PR TITLE
Updated with new UtilityAPI python software engineering job

### PR DESCRIPTION
**Pull request by daniel-utilityapi**
We are hiring another python software engineer, so I updated our listing with a new timestamp and description. Thanks!

### CODE REVIEW

File: jobs/utilityapi_software_engineer.html [hunk 1]
```diff
@@ -17,33 +17,40 @@
   - united-states
```
Comment (salimfadhley) [inline jobs/utilityapi_software_engineer.html]: United Staes shouldn't be a tag. This field is reserved for technical skills required for the job.

File: jobs/utilityapi_software_engineer.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     name: Daniel Roesler
```
Comment (salimfadhley) [inline jobs/utilityapi_software_engineer.html]: Also, I missed this in the previous review, but can you put a more precise location in the Location field - unless this is a 100% remote working job we want to see at least the city & state.

Comment (daniel-utilityapi) [reply jobs/utilityapi_software_engineer.html]: Yes, this is a 100% remote working job in the United States. Should I keep it as United States or put our central office (Oakland, CA) even if they don't have to work there?

File: jobs/utilityapi_software_engineer.html (hunk 1 above)
Comment (daniel-utilityapi) [reply jobs/utilityapi_software_engineer.html]: Fixed!

**Comment by daniel-utilityapi** [comment]
@salimfadhley pushed some updates to satisfy your requests, please let me know if you need me to change anything else!

**Comment by salimfadhley** [comment]
I am going to merge this now, but I think it would be better to revise it and say that the location is Oakland, California but that 100% remote working is an option.

**Pull request merged**